controllers/release/adapter_test.go

```diff
@@ -935,6 +872,170 @@ var _ = Describe("Release Adapter", Ordered, func() {
 		})
 	})
 
+	Context("When registerDeploymentData is called", func() {
+		var adapter *Adapter
+
+		AfterEach(func() {
+			_ = adapter.client.Delete(ctx, adapter.release)
+		})
+
+		BeforeEach(func() {
+			adapter = createReleaseAndAdapter()
+		})
+
+		It("does nothing if there is no binding", func() {
+			Expect(adapter.registerDeploymentData(nil, nil)).To(Succeed())
```

Comment:
should be passing `nil, releasePlanAdmission`

Reply:
Yeah, that's better actually.